source/MRMesh/MRSerializer.h

```diff
@@ -146,6 +146,8 @@ MRMESH_API void deserializeFromJson( const Json::Value& root, TriPointf& tp );
 MRMESH_API void deserializeFromJson( const Json::Value& root, MeshTriPoint& mtp, const MeshTopology& topology );
 MRMESH_API void deserializeFromJson( const Json::Value& root, PointOnFace& pf );
 
+MRMESH_API tl::expected<std::shared_ptr<Object>, std::string> deserializeObjectTreeFromGltf( const std::filesystem::path& file, ProgressCallback callback = {} );
```

Comment:
please add comment about this function

Reply:
also mb move it to separate file? this file is mostly for serialization and deserialization of simple structs